src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -431,6 +431,22 @@ module Types = struct
       ~doc:"Status for whenever the blockchain is reorganized"
       ~values:[enum_value "CHANGED" ~value:`Changed]
 
+  let protocol_amounts =
+    obj "ProtocolAmounts" ~fields:(fun _ ->
```

Comment:
Can you rename to `GenesisConstants` for future proofing

Reply:
In the code that produces these values, the name is `constraint constants`.  Shouldn't there be one name, everywhere?

Reply:
The type is `Genesis_constants.Constraint_constants.t`, conceptually supposed to be the part of the genesis constants where modifications will affect the snark constraint system. This is useful for knowing what can be modified (eg. through the runtime config), but I don't think we need that level of granularity here.

Reply:
:+1: